### DISCUSSION TITLE
Suggestion: Countdown timer.

### DISCUSSION THREAD

**Discussion by OttoMan28** [arendst/Tasmota#13660] [Q&A] [answered]
I think it could be beneficial to have a countdown time, e.g. if you have a device you only use now and then, you could set it to x hours and you’ll know it would be turned off (or on) automatically.

Lets say you have a heater in your workshop, and you know that you have to spend a coupe of hours on a project in there, then you set it to e.g. 4 hours and forget about it.

/Otto

**Comment by ascillato** [comment, marked as answer]
Already implemented. Please, check for pulsetimer command in Tasmota docs. thanks.

**Reply by sfromis** [reply]
If the heater should not be on full time while the workshop is inuse, you can create rules with a RuleTimer to ensure that it will be turned off after a set period.

**Comment by OttoMan28** [comment]
Hi Guys

I was not aware of that, it works great - thanks.
But i was more in the line of a direct function on the web interface, so the timeout value could change on the fly.
```
[     Toggle      ]
[ Countdown timer ]
[   Information   ]
```
and so on.

But for the pulsetimer, i have some other greate use for it :-)
TIA
